### PR TITLE
Staging to main

### PR DESCRIPTION
**Pull request by Nikhil-Fexle-May2022**

### CODE REVIEW

File: SRC/package.xml [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
-		<members>StudentQuery</members>
+		    <members>HelloWorld</members>
+		    <members>StudentQuery</members>
         <members>Teacher</members>
         <name>ApexClass</name>
     </types>
     <types>
         <members>Dream_World__c.Tesla__c</members>
-		<members>Hello_World__c.Hello__c</members>
+		    <members>Hello_World__c.Hello__c</members>
```
Comment (deekshithr2694) [inline SRC/package.xml:12]: Start to use the indentation correctly make it as practice

File: SRC/classes/StudentQuery.cls [hunk 2]
```diff
@@ -0,0 +1,10 @@
+public class StudentQuery {
+    public StudentQuery(){
+        List<Student__c> StudentList = [select id, Name,Roll_Number__c,Father_Name__c,Father_Mobile_Number__c,
```
Comment (deekshithr2694) [inline SRC/classes/StudentQuery.cls:3]: Always Use Query Indentation 
Ex: Select or SELECT it shouldn't be select, And all field names should be start with capital Letters Id, Name and Variables camelCase ex: studentList

File: SRC/classes/StudentQuery.cls [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+                                        HomePhone__c,Email__c,Address__c,Overall_Grade__c
+                                        from Student__c order by createdDate Limit 5];
+        	for(Student__c Teacher: StudentList){
+            	System.debug('Check data ::: '+Teacher);
```
Comment (deekshithr2694) [inline SRC/classes/StudentQuery.cls:7]: While Adding the code changes for PR please review that no debug statements are present.

File: SRC/classes/Teacher.cls [hunk 4]
```diff
@@ -0,0 +1,42 @@
+public class Teacher {
+    public Teacher(){
+    // ################## Student Personal Information #################
+    Student__c Student = new Student__c();
```
Comment (deekshithr2694) [inline SRC/classes/Teacher.cls:4]: use camel case for variables

File: SRC/package.xml (hunk 1 above)
Comment (Nikhil-Fexle-May2022) [reply SRC/package.xml]: Okay sir

**Review by deekshithr2694 — APPROVED** [review]

File: SRC/classes/Teacher.cls (hunk 4 above)
Comment (deekshithr2694) [reply SRC/classes/Teacher.cls:4]: Create a new PR for above Comments

**Pull request merged**